runtime-testsuite/test/org/antlr/v4/test/runtime/go/BaseTest.java

```diff
@@ -391,7 +391,7 @@ public String execModule(String fileName) {
 			ProcessBuilder builder = new ProcessBuilder(goPath, "run", modulePath,
 					inputPath);
 			builder.environment().put("GOPATH",
-					runtimePath + File.pathSeparator + tmpdir);
+					builder.environment().get("GOPATH") + ":" + runtimePath + File.pathSeparator + tmpdir);
```

Comment:
Good question. I'm not familiar with the API, I actually guessed there's a `get` method based on the use of the `set` method. Open to suggestions.